person.rb

```diff
@@ -7,10 +7,11 @@ def initialize(age, name = 'Unknown', parent_permission: true)
     @age = age
     @name = name
     @parent_permission = parent_permission
+    @rentals = []
   end
 
   attr_accessor :name, :age
-  attr_reader :id
+  attr_reader :id, :rentals
```

Comment:
- [ ] Getters and Setters for @rentals should be present, Kindly add this ✍🏻 

- [ ]  A method for adding rentals should be present. Kindly add this 👍🏻 and make sure The person itself is sent as a parameter to create the new rental achieving the 'has-many' association


```ruby
  def add_rental(...)
 ...
  end
```